src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.CitizenDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.Citizen;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+    @Inject
+    CitizenDao citizenDao;
+
+    public void handleCitizen() {
+        System.out.println("Please, add name and age! ");
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Citizen citizen = null;
```

Comment:
There is no need to initialize the variable. In the case of inputting the correct data - your variable will be initialized, else you ask the user to try again or quit the service.